scipy/optimize/_qap.py

```diff
@@ -507,6 +519,29 @@ def _quadratic_assignment_faq(A, B,
     return OptimizeResult(res)
 
 
+def _adj_pad(A, B, method):
+    # pads the matrix with less nodes such that A & B are same size
+    # schemes according to section 2.5 of [2]
+    def pad(X, n):
+        X_pad = np.zeros((n[1], n[1]))
+        X_pad[: n[0], : n[0]] = X
+        return X_pad
+
+    A_n = A.shape[0]
+    B_n = B.shape[0]
+    n = np.sort([A_n, B_n])
+    if method == "adopted":
```

Comment:
![image](https://user-images.githubusercontent.com/6570539/98703449-59631480-2330-11eb-9d64-ade09837915e.png)

Should the n's in the paper have subscripts?